its/src/test/java/com/sonar/it/scanner/msbuild/utils/BuildCommand.java

```diff
@@ -55,12 +56,20 @@ public BuildCommand useDotNet(String dotnetCommand) {
     return this;
   }
 
+  public BuildCommand withExplicitRestore() {
```

Comment:
Which restore? `nuget restore`, `msbuild /t:Restore`, or `dotnet restore`? This should be something like `withNuGetRestore` or `useNuGetRestore`